source/Tests/Infrastructure/OutgoingMessages/RSM012/AssertMeteredDataForMeteringPointEbixDocument.cs

```diff
@@ -0,0 +1,297 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Globalization;
+using Energinet.DataHub.EDI.BuildingBlocks.Domain.Models;
+using Energinet.DataHub.EDI.OutgoingMessages.Domain.DocumentWriters.Formats.Ebix;
+using Energinet.DataHub.EDI.Tests.Infrastructure.OutgoingMessages.Asserts;
+using Energinet.DataHub.EDI.Tests.Infrastructure.OutgoingMessages.NotifyWholesaleServices;
+using FluentAssertions;
+using NodaTime;
+using NodaTime.Text;
+
+namespace Energinet.DataHub.EDI.Tests.Infrastructure.OutgoingMessages.RSM012;
+
+public class AssertMeteredDataForMeteringPointEbixDocument : IAssertMeteredDataForMeteringPointDocumentDocument
+{
+    private readonly AssertEbixDocument _documentAsserter;
+
+    public AssertMeteredDataForMeteringPointEbixDocument(
+        AssertEbixDocument documentAsserter,
+        bool skipIdentificationLengthValdation = false)
+    {
+        _documentAsserter = documentAsserter;
+        _documentAsserter.HasValue("HeaderEnergyDocument/DocumentType", "E66");
+    }
+
+    public IAssertMeteredDataForMeteringPointDocumentDocument HasBusinessReason(string expectedBusinessReasonCode)
+    {
+        _documentAsserter.HasValueWithAttributes(
+            "ProcessEnergyContext/EnergyBusinessProcess",
+            expectedBusinessReasonCode,
+            CreateRequiredListAttributes(expectedBusinessReasonCode.StartsWith('D') ? CodeListType.EbixDenmark : CodeListType.Ebix));
+        return this;
+    }
+
+    public IAssertMeteredDataForMeteringPointDocumentDocument HasBusinessSectorType(string? expectedBusinessSectorType)
+    {
+        if (expectedBusinessSectorType is null)
+        {
+            _documentAsserter.IsNotPresent("ProcessEnergyContext/EnergyIndustryClassification");
+        }
```

Comment:
```suggestion
        if (expectedBusinessSectorType is null)
        {
            _documentAsserter.IsNotPresent("ProcessEnergyContext/EnergyIndustryClassification");
        }
```
Eventho the schema allows it, i would argue that "expectedBusinessSectorType" == null is not a valid scenario